.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+<!--
+    Thank you very much for contributing to this project by creating an issue!
+    To avoid duplicate issues we ask you to check off the following list.
+-->
+
+- [ ] I have searched the [issues](https://github.com/opencv/training_toolbox_tensorflow/issues) of this repository and believe that this is not a duplicate.
+
+## Expected Behavior
+    If you're describing a bug, tell us what should happen.
+    If you're suggesting a change/improvement, tell us how it should work.
+
+## Current Behavior
+    If describing a bug, tell us what happens instead of the expected behavior.
+    If suggesting a change/improvement, explain the difference from current behavior.
+
+## Steps to Reproduce (for bugs)
+
+1.
+2.
+3.
+4.
+
+## Context
+    How has this issue affected you? What are you trying to accomplish?
+    Providing context helps us come up with a solution that is most useful in the real world.
+
+## Your Environment
+    Include as many relevant details about the environment with which you experienced the bug.
```

Comment:
It can be irrelevant, but yes :)

Reply:
Why not to ask for relevant information explicitly ? (e.g. TF version, Python version, etc.)

Reply:
I don't want to spend time on that list right now, because we will form an incomplete list anyway.
I believe we should wait for the first submitted bug and update this part of the template after it.

Reply:
That's exactly my point. Instead of adding generally reasonable instructions on how to use git and submit bugs, we can simply wait for the problem to reveal itself and make the instructions address the specific points that are relevant. 

On this specific point - even an incomplete list is better than "include everything you can"

Reply:
Added, but "include everything you can" and "include as many relevant details" are not the same.

Reply:
It's practically the same